spec/ddtrace/profiling_spec.rb

```diff
@@ -1,6 +1,6 @@
 # typed: false
```

Comment:
⬆️  this file should also be moved to `spec/datadog/profiling_spec.rb`

Reply:
Done!